src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1382,15 +1407,13 @@ internal enum ErrorCode
         ERR_BadIteratorLocalType = 8941,
         ERR_BadAsyncLocalType = 8942,
         ERR_RefReturningCallAndAwait = 8943,
+
+        #endregion diagnostics for ref locals and ref returns introduced in C# 7
+
+        #region diagnostics introduced for original/replace in C# 7
         ERR_NoOriginalMember = 8944,
         ERR_DuplicateReplace = 8945,
         ERR_PartialReplace = 8946,
-        ERR_TupleTooFewElements = 8200,
-        ERR_TupleReservedMemberName = 8201,
-        ERR_TupleReservedMemberNameAnyPosition = 8202,
-        ERR_TupleDuplicateMemberName = 8203,
-        ERR_TupleExplicitNamesOnAllMembersOrNone = 8204,
-
-        ERR_PredefinedTypeMemberNotFoundInAssembly = 8205,
+        #endregion diagnostics introduced for original/replace in C# 7
```

Comment:
The previous ranges had comments instead of regions that suggested that they were introduced in stages. I'm just documenting them in a more useful way, and being consistent with the new ones. Since the new errors are not in numerical order today, it is very difficult to find where there are gaps, thus the reorganization here.

---
In reply to: [64667315](https://github.com/dotnet/roslyn/pull/11485#discussion_r64667315) [](ancestors = 64667315)